free-programming-books-zh.md

```diff
@@ -609,9 +609,12 @@
 ### Python
 
 * [Django book 2.0](http://djangobook.py3k.cn/2.0/)
+* [Python 3 Module of the Week](https://pymotw.com/3/#python-3-module-of-the-week)
 * [Python 3 文档(简体中文) 3.2.2 documentation](http://docspy3zh.readthedocs.org/en/latest/)
+* [Python Cookbook第三版](http://python3-cookbook.readthedocs.io/zh_CN/latest/)
```

Comment:
This book translate from《Python Cookbook》3rd Edition，in this case, should I add authors or translator